_includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-feature.md

```diff
@@ -563,6 +563,7 @@ class PickingListDependencyProvider extends SprykerPickingListDependencyProvider
 | PLUGIN                                                          | SPECIFICATION                                                                              | PREREQUISITES | NAMESPACE                                                                                          |
 |-----------------------------------------------------------------|--------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------|
 | PickingListsBackendResourcePlugin                               | Registers the `picking-lists` resource.                                                    |               | Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication                               |
+| PickingListStartPickingBackendResourcePlugin                    | Registers the `picking-lists` resources `start-picking` resource.                          |               | Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication                               |
```

Comment:
this looks wrong
we have  different features mentioned in the singe IG

See  [Warehouse picking](https://release.spryker.com/features/view/140) and [Push Notification](https://release.spryker.com/features/view/139)
I see some picking list plugin was added to this IG by mistake
Please update different IGs respectively

Reply:
Please clarify what exactly is looking wrong in your opinion. Everything related to the warehouse picking is mentioned within [install-the-warehouse-picking-feature.md](https://github.com/spryker/spryker-docs/pull/1951/files/cd23094b82bf63bd5fc5c2137082ab2ae729f636#diff-27e352da16d1c597b6b4990863a44369504b0a5e5c306111b33ffd0c0f3183d7) and everything related to push notifications is mentioned in the other. These got just updated, so i don't see any issue.